deploy/gunicorn_start.sh

```diff
@@ -2,7 +2,7 @@
 
 NAME="memex_explorer"                                  # Name of the application
 DJANGODIR=/vagrant/source             # Django project directory
-SOCKFILE=/tmp/gunicorn_supervisor.sock # we will communicte using this unix socket
+SOCKFILE=/home/vagrant/gunicorn.sock # we will communicte using this unix socket
```

Comment:
Upon reflection, I actually wonder if having all of these in /home/vagrant isn't still too messy.